Tribler/community/trustchain/block.py

```diff
@@ -13,21 +14,20 @@
 EMPTY_SIG = '0'*SIG_LENGTH
 EMPTY_PK = '0'*PK_LENGTH
 
-block_pack_format = "! Q Q Q Q {0}s I {0}s I {1}s {2}s".format(PK_LENGTH, HASH_LENGTH, SIG_LENGTH)
+block_pack_format = "! {0}s I {0}s I {1}s {2}s".format(PK_LENGTH, HASH_LENGTH, SIG_LENGTH)
 block_pack_size = calcsize(block_pack_format)
 
 
-class MultiChainBlock(object):
+class TrustChainBlock(object):
     """
-    Container for MultiChain block information
+    Container for TrustChain block information
```

Comment:
Please add some documentation on the usage of this class. In particular I would like to know how to add my own transaction data.